### DISCUSSION TITLE
Checkout System?

### DISCUSSION THREAD

**Discussion by b3ck** [gollum/gollum#1996] [Q&A] [answered]
Is there a checkout system?

Like can two people edit the same page at the same time?

I hope not.

**Comment by bartkamphorst** [comment, marked as answer]
Yes, two people can edit the same page at the same time, and yes, this can result in conflicts. At present, when users encounter an edit conflict, they are notified and asked to save their work offline and integrate their edits into the new version manually. This isn't ideal, so we have [plans](https://github.com/gollum/gollum/pull/1989) to offer a merge tool for when this occurs.

**Comment by b3ck** [comment]
Copy that, thank you 👍
